### PR TITLE
Liquidation Recovery Config Automation

### PR DESCRIPTION
**Pull request by beaushinkle**
This PR adds automation to the local `./install.sh` script to populate the keep-ecdsa configs with the relevant configuration bits for liquidation recovery. This allows developers to run a fully configured node for liquidation recovery. This will be mostly useless until either the liquidation-recovery-integration branch is merged in `keep-ecdsa` or if a developer is manually checking out that branch for local builds.

Tagging @nkuba for review

### CODE REVIEW

File: install-keep-ecdsa.sh [hunk 1]
```diff
@@ -9,6 +9,10 @@ DONE_END='\n\n\e[0m'    # new line + reset
 
 WORKDIR=$PWD
 
+printf "${LOG_START}Installing bitcoind-wallet...${LOG_END}"
+
+npm install -g "$WORKDIR/bitcoind-wallet"
```
Comment (nkuba) [inline install-keep-ecdsa.sh:14]: What do you think about pulling this part to a separate script `initialize-bitcoin.sh`? We already have `initialize-geth.sh`. We could mention the need to run the init script in `manual-setup.adoc`.

Comment (beaushinkle) [reply install-keep-ecdsa.sh:14]: Just the bitcoind-wallet install, or the config changes as well?

If it's just the bitcoind-wallet stuff, that works for me!

Comment (nkuba) [reply install-keep-ecdsa.sh]: Yes, just the bitcoind-wallet install!

**Review by nkuba — APPROVED** [review]
:fire:

**Pull request merged**